pkg/gofr/datasource/cassandra/errors.go

```diff
@@ -1,17 +1,23 @@
 package cassandra
 
-import "fmt"
+import (
+	"errors"
+	"fmt"
+)
 
 const (
 	msgDestinationIsNotPointer = "destination is not pointer"
-	msgUnexpectedMap           = "a map was not expected."
+	msgUnexpectedMap           = "a map was not expected"
+	msgUnsupportedBatchType    = "batch type not supported"
+	msgBatchNotInitialised     = "batch not initialized"
 )
 
-type destinationIsNotPointer struct{}
-
-func (d destinationIsNotPointer) Error() string {
-	return msgDestinationIsNotPointer
-}
+var (
+	errDestinationIsNotPointer = errors.New(msgDestinationIsNotPointer)
+	errUnexpectedMap           = errors.New(msgUnexpectedMap)
+	errUnsupportedBatchType    = errors.New(msgUnsupportedBatchType)
+	errBatchNotInitialised     = errors.New(msgBatchNotInitialised)
+)
 
 type unexpectedPointer struct {
```

Comment:
Same remark for this one and unexpectedSlice

I would expect them to be exported